### ISSUE TITLE
Request for Detailed Code Steps for Breast Edge Detection and Normalization Using OpenCV on CBIS-DDSM Dataset

### ISSUE THREAD

**Issue by sunxuan1013** [dpetrini/two-views-classifier#2]
Hi there,

First of all, thank you for your great work on this project. It has been very helpful for my research.

I’m a beginner working with the CBIS-DDSM dataset and I’m trying to use OpenCV functions for breast edge detection and masking, as well as cropping out the black background and normalizing the images. However, the results I’m getting don’t seem quite right.

Could you kindly provide more detailed steps or any code snippets for this process? I’d really appreciate any guidance you can offer.

Thank you so much for your time and help!

**Comment by dpetrini** [comment]
Hello,
What do you need exactly, he image Processing Pipeline  in dataloaling process?

**Comment by sunxuan1013** [comment]
> 您好， 您到底需要什么，数据加载过程中的图像处理管道？

Yes, in the data processing pipeline, your method involves breast boundary detection, masking, cropping the black background, and normalizing the images. However, with such a large number of images in this dataset, handling them one by one can be incredibly overwhelming. If not processed correctly, it might also impact the experimental results.

**Comment by dpetrini** [comment]
Hello,

I added some processing steps here:  https://github.com/dpetrini/prepare_datasets/

Please wait for more commits of remaining pipeline soon.

BR

**Comment by keny-phong** [comment]
Hello D Petrini, could you tell me why you work on this project ?

**Comment by dpetrini** [comment]
> Hello D Petrini, could you tell me why you work on this project ?


Hello, I worked with academic purposes.

We are piblishing a paper with this work.

Ler me l now if you need more information.

Br

**Comment by keny-phong** [comment]
Hi, thank you for your reply.

After reading few subject about cancer none of them talked about prevention
and detection.

I got my cat who got cancer and I would like to save the others cats /
dogs.

Next month I will work on breast cancer for human and I did not see any
research on cancer for animals.

I think the cancers for human or animals should be similar. If you have any
paper on this subject I will appreciate if you could provide it ?

Regards.

K.T. PHONGSAVANH

**Comment by keny-phong** [comment]
Hi again,

I make a mistake for the destination for the first mail.

I am about to work on a project on breast cancer with a physician and
developper next month.

I saw that you put some codes on github. But, I dont know if I can use it
as it is.

If you have a complete version of the application, I will appreciate if you
can help us.

Basically, you can see my profile ont linkedin.

Thank you

K.T. PHONGSAVANH
